example/tsconfig.json

```diff
@@ -25,5 +25,6 @@
     "noFallthroughCasesInSwitch": true
   },
   "include": ["src"],
+  "exclude": ["src/email"],
```

Comment:
May be here should be proposed more comrehensive solution to have ts checking in both projects? Just disabling doesn't seem right to me. 

For example this folder may have a separate tsconfig, with its own settings and being referenced from a root tsconfig. Or  keep this folders separated at all as it was originally

Reply:
You're right, there's really no reason to disable type checking.  
TypeScript doesn't actually transpile those files; it only uses them to surface errors during the build process.  
And even though the email code targets a very different environment than the rest of the app, I think it's fine in practice to use the same `tsconfig` for both the emails and the browser-targeted code.